web/client/components/import/style/StylePanel.jsx

```diff
@@ -0,0 +1,198 @@
+const PropTypes = require('prop-types');
```

Comment:
Is this another duplicate of shapefile related code?

Reply:
Yes, separated the styling functionalities from the shape file, so we can replace it in the future with a better styler. Same as above, didn't want to update a component that may be removed